### PR TITLE
Ignore stacktrace part of error screen to be screenshot

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

fix flawkyness of playwright tests regarding the OS unsupported test.

### ❓ Context

- **Impacted projects**: `LLD` 
- **Linked resource(s)**: `n/a` 

### ✅ Checklist

- [x] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

**Review by nabil-brn — APPROVED** [review]

File: apps/ledger-live-desktop/tests/specs/support/unsupported-os.spec.ts [hunk 1]
```diff
@@ -15,6 +15,10 @@ test("Unsupported OS", async ({ page }) => {
   await test.step("displays the error page", async () => {
     await layout.renderError.waitFor({ state: "visible" });
     await expect(layout.renderError).toBeVisible();
-    await expect(await layout.page.screenshot()).toMatchSnapshot("error-os-unsupported.png");
+    await expect(
+      await layout.page.screenshot({
+        mask: [page.locator("[data-test-id='stacktrace']")],
```
Comment (Justkant) [inline apps/ledger-live-desktop/tests/specs/support/unsupported-os.spec.ts:20]: Shouldn't the `page.locator` live in the `Layout` model ?

Comment (nabil-brn) [reply apps/ledger-live-desktop/tests/specs/support/unsupported-os.spec.ts:20]: No, it comes from `tests/fixtures/common.ts` and `@playwright/test` package.

Comment (Justkant) [reply apps/ledger-live-desktop/tests/specs/support/unsupported-os.spec.ts:20]: That doesn't answer my question, we're suppose to put all `page.locator` calls in the constructor of a model, I saw that everywhere, why not for this `page.locator` ?

Comment (nabil-brn) [reply apps/ledger-live-desktop/tests/specs/support/unsupported-os.spec.ts:20]: My bad, I didn’t understand that you were mentioning this line, of course locators should live in the page object, it’s a good practice.

**Comment by gre** [comment]
if this issue is a problem again, someone will need to find a solution but this PR didn't address the fact the UI is randomly stretching with different width and it seems that playwright selector approach won't work because the "dark square" that replace the content will change of width. 

closing this PR because i don't have time to dig more, feel free to reopen if it is a new issue to fix in future

**Pull request closed without merging**